client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.jsx

```diff
@@ -155,52 +204,94 @@ const DataFilesBreadcrumbs = ({
       }
     }, '');
 
+  const lastTwoPaths = paths.slice(-1);
+  const lastTwoPathComps = pathComps.slice(-1);
+  const reversedPath = paths.reverse();
+
   return (
-    <div className={`breadcrumbs ${className}`}>
-      {scheme === 'projects' && (
-        <>
-          <RootProjectsLink
-            api={api}
-            section={section}
-            operation={operation}
-            label="Shared Workspaces"
-          />{' '}
-          {system && `/ `}
-        </>
+    <div className="breadcrumb-container">
+      <div id="path-button-wrapper">
+        <ButtonDropdown
+          isOpen={dropdownOpen}
+          toggle={toggleDropdown}
+          className="go-to-button-dropdown"
+        >
+          <DropdownToggle
+            color="secondary"
+            id="data-files-path"
+            className="data-files-btn"
+          >
+            Go to ...
+          </DropdownToggle>
```

Comment:
The font does not match the other buttons. And the `data-btn-styles` appear to be a new attempt to mimic existing styles. Please investigate how to use a `<Button>` as the dropdown toggle (so that button styles have one source of truth).

Reply:
I will look into using the Button component for the toggle

Reply:
<img width="1428" alt="Screenshot 2023-09-29 at 3 33 46 PM" src="https://github.com/TACC/Core-Portal/assets/50084480/d4633bbd-3d31-428b-84b1-1fe9362f75b1">
<img width="1428" alt="Screenshot 2023-09-29 at 3 33 58 PM" src="https://github.com/TACC/Core-Portal/assets/50084480/35d48935-f2c4-47c7-823f-e973e8b5c319">